### ISSUE TITLE
Noisy logs during play

### ISSUE THREAD

**Issue by muchcharles** [ValveSoftware/steamvr_unreal_plugin#91]
I'm seeing a lot of logging during play with the new integration of the plugin that is included in 4.24:

```
Sat Nov 09 2019 17:04:35.123 - /filters/thumbstick_settings__actions_main_7_user_hand_left_input_thumbstick/x -- ReadScalarValue returned 0, time=2459784264122, origin=/user/hand/left/input/thumbstick/x
Sat Nov 09 2019 17:04:35.123 - /filters/thumbstick_settings__actions_main_7_user_hand_left_input_thumbstick/y -- ReadScalarValue returned 0, time=2459784264122, origin=/user/hand/left/input/thumbstick/y
```

This seems to only go to stdout or stderr (visible in visual studio output window when running but not in unreal log file, and not using unreal's log formatting).  I looked through and didn't see anything setting k_pch_SteamVR_LogLevel_Int32  within the unreal plugins, so it may just be that the default log level within steamvr itself is being too noisy?

**Comment by 1runeberg** [comment]
Hi @muchcharles - thanks for the report/feedback.

If you need less verbose logging, you can set "loglevel" to 2 or 1 in your:

/Program Files (x86)/Steam/config/**steamvr.vrsettings**

`
"steamvr" : {
       ...
      "loglevel" : 2,
      ...
   }
`

**Comment by 1runeberg** [comment]
Closing as this is the default from the runtime.

**Comment by muchcharles** [comment]
May want to re-open this one, seems the issue I reported here 

https://github.com/ValveSoftware/openvr/issues/1271 is actually still caused by this and this line (which is now in the module startup as of 4.24.1):

    VRSettings()->SetBool(k_pch_SteamVR_Section, k_pch_SteamVR_DebugInput, true, &BindingFlagError);

**Comment by JoeLudwig** [comment]
@1runeberg will pull that from the plugin. 

That setting should really have been called "joesSpammyDebugSetting". It's nearly useless for anyone other than me when I'm debugging some minutia of the input system guts. So I'm going to rename it to something along that line and remove it from openvr.h. That should fix anyone who has the setting set because of the plugin.

**Comment by 1runeberg** [comment]
Added to PR for 4.24.2: https://github.com/EpicGames/UnrealEngine/pull/6459

**Comment by 1runeberg** [comment]
Hi @muchcharles - can you confirm on your end with 1.9.15 that this isnt an issue anymore?

Thanks,
Rune

**Comment by muchcharles** [comment]
> Hi @muchcharles - can you confirm on your end with 1.9.15 that this isnt an issue anymore?
> 
> Thanks,
> Rune

Seems fixed, I tried with the launcher 4.24 build and didn't see the output or large logs anymore.

**Comment by kisak-valve** [comment]
Thanks for retesting, closing.
